src/binding.cc

```diff
@@ -123,18 +103,20 @@ struct Iconv
     }
     input_consumed -= input_size;
     output_consumed -= output_size;
-    rc->Set(0, Nan::New<Integer>(static_cast<uint32_t>(input_consumed)));
-    rc->Set(1, Nan::New<Integer>(static_cast<uint32_t>(output_consumed)));
-    info.GetReturnValue().Set(errorno);
+    rc.Set(static_cast<uint32_t>(0), Napi::Number::New(env, static_cast<uint32_t>(input_consumed)));
```

Comment:
For my edification, this `static_cast<uint32_t>(0)` is necessary because `rc.Set(0, ...)` is ambiguous w.r.t. to the overload that takes a char pointer?

Reply:
It's necessary because you can only pass this types to `Set` method:
- `napi_value`
- `Napi::Value`
- `const char*`
- `const std::string&`
- `uint32_t `
in alternative it could be possible do the same thing like reported below:
```cpp
rc.Set(Napi::Number::New(env, 0), Napi::Number::New(env, static_cast<uint32_t>(input_consumed)));
rc.Set(Napi::Number::New(env, 1), Napi::Number::New(env, static_cast<uint32_t>(output_consumed)));
```
What do you prefer?